### PR TITLE
Fix voucher type input

### PR DESCRIPTION
**Pull request by Irene350**

### CODE REVIEW

**Comment by Irene350** [comment]
Hi @darekg11 can you have a look at this? thanks

File: packages/sdk/src/types/Vouchers.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { OrdersGetResponse } from './Orders'
 import { SimpleCustomer } from './Customers'
 import { DiscountUnit, DiscountAmount, DiscountPercent } from './DiscountVoucher'
 
-export type VoucherType = 'GIFT' | 'DISCOUNT' | 'LOYALTY_CARD' | 'LUCKY_DRAW'
+export type VoucherType = 'GIFT' | 'DISCOUNT_VOUCHER' | 'LOYALTY_CARD' | 'LUCKY_DRAW'
```
Comment (darekg11) [inline packages/sdk/src/types/Vouchers.ts:5]: Dear @Irene350 could you also change `GIFT` to `GIFT_VOUCHER`? Unfortunately we overlook this when doing first iteration for this SDK.

Comment (Irene350) [reply packages/sdk/src/types/Vouchers.ts:5]: sure thanks for double checking!

**Review by darekg11 — APPROVED** [review]

**Comment by darekg11** [comment]
Dear @Irene350 thank you for yet another contribution.
I have just released `2.1.0` version and it should be available in NPM shortly.

**Pull request merged**